oshi-core/src/main/java/oshi/hardware/platform/unix/aix/AixCentralProcessor.java

```diff
@@ -121,15 +123,48 @@ protected Pair<List<LogicalProcessor>, List<PhysicalProcessor>> initProcessorCou
         if (lcpus < 1) {
             lcpus = 1;
         }
+        int lpPerPp = lcpus / physProcs;
```

Comment:
it seems that `lpPerPp` can become 0... eg. 1/2, 1/3... and then cause a divide by zero on line 132.

see #2228
explainer https://stackoverflow.com/questions/4685450/int-division-why-is-the-result-of-1-3-0

Reply:
> explainer https://stackoverflow.com/questions/4685450/int-division-why-is-the-result-of-1-3-0

Better explainer:  https://xkcd.com/2295/